test/functional/wallet_labels.py

```diff
@@ -156,7 +156,7 @@ def run_test(self):
             ad = BECH32_INVALID[l]
             assert_raises_rpc_error(
                 -5,
-                "Invalid Bitcoin address or script",
+                "Address is not valid" if self.options.descriptors else "Invalid Bitcoin address or script",
                 lambda: wallet_watch_only.importaddress(label=l, rescan=False, address=ad),
             )
 
```

Comment:
in commit c1f7bfc6bb1074c26d670976a8f8c0bb95be1858:

Should wallet_labels be run with and without descriptors in the test_runner?